homeassistant/components/home_connect/__init__.py

```diff
@@ -244,7 +244,7 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 
 @Throttle(SCAN_INTERVAL)
```

Comment:
We want to remove as much throttles as possible, so it would be interesting to explore the option of adding a coordinator

Reply:
Maybe we can create an issue to add this to the backlog, because doing this change in this PR will take time and will slow this PR and the other pending PRs.